src/cli/cli.js

```diff
@@ -90,6 +90,7 @@ async function runTests (argParser) {
         configFile,
         disableHttp2,
         v8Flags,
+        fromCli: true,
```

Comment:
A runner already has such a flag.
Use the naming from the runner and pass this option to the runner too.
https://github.com/DevExpress/testcafe/blob/2d536f3dfedba782086fb21d15148878147d0510/src/runner/index.js#L64